C/plugins/north/OMF/plugin.cpp

```diff
@@ -356,6 +389,14 @@ const char *PLUGIN_DEFAULT_CONFIG_INFO = QUOTE(
 			"default": NOT_BLOCKING_ERRORS_DEFAULT_PI_WEB_API,
 			"order": "28" ,
 			"readonly": "true"
+		},
+		"Legacy": {
+			"description": "Force all data to be sent using complex OMF types",
+			"type": "boolean",
+			"default": "false",
+			"order": "29",
+			"group": "Formats & Tyes",
+			"displayName": "Complex Types"
```

Comment:
Will it be clear to the system administrator that his/her original OMF configuration is considered a "complex type?" In the description, "Force" seems a bit strong. I can't think of better terms for either, though.